index.bs

```diff
@@ -461,31 +461,33 @@ order to reduce the fingerprint the header provides.
 GREASE-like UA Strings {#grease}
 ----------------------
 
-History has shown us that there are real incentives for [=user agents=] to lie
-about their branding in order to thread the needle of sites' sniffing scripts,
-and prevent their users from being blocked by UA-based allow/block lists.
-
-Reseting expectations may help to prevent abuse of the UA string's brand in the
-short term, but probably won't help in the long run.  The world of network
-protocol introduced the notion of <abbr title="Generate Random Extensions And
-Sustain Extensibility">GREASE</abbr> [[I-D.ietf-tls-grease]]. We could borrow
-from that concept to tackle this problem.
-
-[=User agents=] SHOULD model `UA` as a set, rather than a single entry. They then
-could encourage standardized processing of the `UA` string, by randomly
-including additional, intentionally incorrect, comma-separated entries with
-arbitrary ordering. That would reduce the chance that we ossify on a few
-required strings.  
+History has shown us that there are real incentives for [=user agents=] to lie about their branding
+in order to thread the needle of sites' sniffing scripts, and prevent their users from being blocked
+by UA-based allow/block lists.
+
+Reseting expectations may help to prevent abuse of the UA string's brand in the short term, but
+probably won't help in the long run.  The world of network protocol introduced the notion of <abbr
+title="Generate Random Extensions And Sustain Extensibility">GREASE</abbr> [[I-D.ietf-tls-grease]].
+We could borrow from that concept to tackle this problem.
+
+[=User agents=] SHOULD model <a http-header>`Sec-CH-UA`</a> and {{uaList}} as a set which contains
```

Comment:
I expected GH to highlight the parts that materially changed, vs. just newline differences. But it didn't. So I'm doing it instead.

This sentence here is the only one including substantive differences. The rest is just newline re-jigging.

Reply:
I think this should instead refer to `[=user agent/UA list=]`, from which both of these things are derived. And "to return the Sec-CH-UA value for a request" should be updated to use UA list, instead of having a separate algorithm.

Finally, the "create UA list" algorithm should change from a MAY to a SHOULD. And then this section should have no normative requirements, but instead talk about the motivation behind the SHOULD in "create UA list".

Reply:
That makes total sense!